pym/bob/input.py

```diff
@@ -576,6 +577,62 @@ def isDeterministic(self):
     def hasJenkinsPlugin(self):
         return True
 
+    def callGit(self, workspacePath, *args):
+        cmdLine = ['git']
+        for arg in args:
+            cmdLine.append(arg)
+        try:
+            output = subprocess.check_output(cmdLine, cwd=os.path.join(os.getcwd(),workspacePath, self.__dir),
+                universal_newlines=True, stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            raise BuildError("git error: '{}' '{}'".format(" ".join(cmdLine), e.output))
+        return output
+
+    def status(self, workspacePath, dir, verbose):
```

Comment:
I would like to introduce at least another level of verbosity:

* 0: don't print anything
* 1: just print that a repo was touched, e.g. "dir: on different commit"
* 2: full details, e.g. with `git status` output what files are dirty

For the status command the default should be 1 because the user typically wants to know if some working copy needs to be taken care of. With the current implementation it is 2. If there are multiple repositories that have changes the output will get confusing.